NineChronicles.Headless/GraphTypes/ActivationStatusMutation.cs

```diff
@@ -64,6 +66,61 @@ public ActivationStatusMutation(NineChroniclesNodeService service)
                         return false;
                     }
 
+                    return true;
+                });
+
+            Field<NonNullGraphType<BooleanGraphType>>("activatePrivateKey",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Name = "encodedActivationKey",
+                    },
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Name = "privateKeyHex",
+                    }
+                ),
+                resolve: context =>
+                {
+                    try
+                    {
+                        string encodedActivationKey =
+                            context.GetArgument<string>("encodedActivationKey");
+                        var privateKeyHex = context.GetArgument<string>("privateKeyHex");
+                        var privateKey = new PrivateKey(ByteUtil.ParseHex(privateKeyHex));
+                        ActivationKey activationKey = ActivationKey.Decode(encodedActivationKey);
+                        if (!(service.Swarm?.BlockChain is { } blockChain))
+                        {
+                            throw new InvalidOperationException($"{nameof(blockChain)} is null.");
+                        }
+
+                        IValue state = blockChain.GetState(activationKey.PendingAddress);
+
+                        if (!(state is Bencodex.Types.Dictionary asDict))
+                        {
+                            context.Errors.Add(new ExecutionError("The given key was already expired."));
+                            return false;
+                        }
+
+                        var pendingActivationState = new PendingActivationState(asDict);
+                        ActivateAccount action = activationKey.CreateActivateAccount(
+                            pendingActivationState.Nonce);
+
+                        var actions = new NCAction[] { action };
+                        blockChain.MakeTransaction(privateKey, actions);
+                    }
+                    catch (ArgumentException ae)
+                    {
+                        context.Errors.Add(new ExecutionError("The given key isn't in the correct format.", ae));
+                        return false;
+                    }
+                    catch (Exception e)
+                    {
+                        var msg = "Unexpected exception occurred during ActivatedAccountsMutation: {e}";
+                        context.Errors.Add(new ExecutionError(msg, e));
+                        return false;
+                    }
+
```

Comment:
This pull request seems for multiple RPC client. Then it looks like the way to make multiple users able to activate without local headless. But IMO it seems too dangerous and too hard to say "give your private key" to users. I wonder other opinions cc @area363 @longfin

Reply:
i agree. so delete this commit and make #676